toolchain/semantics/semantics_node.h

```diff
@@ -153,6 +163,19 @@ class SemanticsNode {
     return SemanticsBuiltinKind::FromInt(arg0_);
   }
 
+  static auto MakeCall(ParseTree::Node parse_node, SemanticsNodeId type,
+                       SemanticsCallId call_id, SemanticsCallableId callable_id)
+      -> SemanticsNode {
+    // Builtins won't have a ParseTree node associated, so we provide the
+    // default invalid one.
```

Comment:
It's not clear to me what relationship this comment has to the surrounding code. Is this comment out of date?